ui/public/navidrome-service-worker.js

```diff
@@ -0,0 +1,48 @@
+// documentation: https://developers.google.com/web/tools/workbox/modules/workbox-sw
+importScripts("https://storage.googleapis.com/workbox-cdn/releases/6.1.2/workbox-sw.js");
```

Comment:
Is this imported at build time or at runtime? I don't want my navidrome instance to phone home to Google.

Reply:
Believe it or not, but the current service worker also does that :)

We can pull in these libraries with npm too during the build.